schemas/threat.yml

```
@@ -24,39 +24,53 @@
 
       example: MITRE ATT&CK
 
-    - name: tactic.name
+    - name: tactic.id
       level: extended
       type: keyword
-      short: Threat tactic.
+      short: Threat tactic id.
       description: >
-          Name of the type of tactic used by this threat. You can use a MITRE ATT&CK® tactic, for example.
-          (ex. https://attack.mitre.org/tactics/TA0040/)
+          The id of tactic used by this threat. You can use a MITRE ATT&CK® tactic, for example.
+          (ex. https://attack.mitre.org/tactics/TA0002/ )
 
-      example: impact
+      example: TA0040
```

```suggestion
      example: TA0002
```

Looks like this should be updated here as well?